airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -24,6 +24,11 @@ application {
 
 }
 
+integrationTestJava {
+    // This is needed to make the destination-snowflake tests succeed - https://github.com/snowflakedb/snowflake-jdbc/issues/589#issuecomment-983944767
+    jvmArgs = ["--add-opens=java.base/java.nio=ALL-UNNAMED"]
+}
+
```

Comment:
Is it related to the secret logic change?

Reply:
No, this is just to trigger a CI run for snowflake.